src/pybind/mgr/dashboard/frontend/src/app/ceph/dashboard/health/health.component.html

```diff
@@ -1,90 +1,49 @@
 <div *ngIf="contentData">
   <div class="row">
     <!-- HEALTH -->
-    <div class="col-md-6">
-      <div class="well">
-        <fieldset>
-          <legend i18n>Health</legend>
-          <ng-container i18n>Overall status: </ng-container>
-          <span [ngStyle]="contentData.health.status | healthColor">{{ contentData.health.status }}</span>
-          <ul>
-            <li *ngFor="let check of contentData.health.checks">
-              <span [ngStyle]="check.severity | healthColor">{{ check.type }}</span>: {{ check.summary.message }}
-            </li>
-          </ul>
-        </fieldset>
-      </div>
-    </div>
+    <cd-info-card title="Health"
+                  class="col-md-6">
+      <hr class="media-underline" />
+      <ng-container i18n>Overall status: </ng-container>
+      <span [ngStyle]="contentData.health.status | healthColor">{{ contentData.health.status }}</span>
+      <ul>
+        <li *ngFor="let check of contentData.health.checks">
+          <span [ngStyle]="check.severity | healthColor">{{ check.type }}</span>: {{ check.summary.message }}
+        </li>
+      </ul>
+    </cd-info-card>
 
     <div class="col-md-6">
       <!--STATS -->
       <div class="row">
-        <div class="col-md-6">
-          <div class="well"
-               *ngIf="contentData.mon_status">
-            <div class="media">
-              <div class="media-left">
-                <i class="fa fa-database fa-fw"></i>
-              </div>
-              <div class="media-body">
-                <span class="media-heading"
-                      i18n="ceph monitors">
-                  <a routerLink="/monitor/">Monitors</a>
-                </span>
-                <span class="media-text">{{ contentData.mon_status | monSummary }}</span>
-              </div>
-            </div>
-          </div>
-        </div>
-        <div class="col-md-6">
-          <div class="well"
-               *ngIf="contentData.osd_map">
-            <div class="media">
-              <div class="media-left">
-                <i class="fa fa-hdd-o fa-fw"></i>
-              </div>
-              <div class="media-body">
-                <span class="media-heading"
-                      i18n="ceph OSDs">
-                  <a routerLink="/osd/">OSDs</a>
-                </span>
-                <span class="media-text">{{ contentData.osd_map | osdSummary }}</span>
-              </div>
-            </div>
-          </div>
-        </div>
+        <cd-info-card title="MONITORS"
```

Comment:
Just a minor comment, but uppercase here would be just a style, so `title` could be 'Monitors' or even 'monitors', and then capitalize it in the component template:

```javascript
{{ title | uppercase }}
```

Reply:
I'm not in favor of forcing the uppercase. In fact, there are design proposals to move to "Monitors".